pytrip/models/proton.py

```diff
@@ -108,20 +107,21 @@ def rbe_mcnamara(dose, let, abx):
     """
 
     _apx = 0.999064 + 0.35605 * let / abx
-    _bpx = 1.1012 - 0.0038703 * np.sqrt(abx) * let
-    _bpx *= _bpx
+    _sbpx = 1.1012 - 0.0038703 * np.sqrt(abx) * let
 
-    rbe = _rbe_apx(dose, _apx, _bpx, abx)
+    rbe = _rbe_apx(dose, _apx, _sbpx, abx)
     return rbe
 
 
-def _rbe_apx(dose, apx, bpx, abx):
+def _rbe_apx(dose, apx, sbpx, abx):
     """
     :params dose: proton dose      [Gy]
     :params apx: alpha_p / alpha_x [dimensionless] RBE_max = ap/ax when (dose -> 0 Gy)
-    :params bpx: beta_p / beta_x   [dimensionless] RBE_min = sqrt(bp/bx) when (dose -> inf Gy)
+    :params sbpx: beta_p / beta_x  [dimensionless] RBE_min = sqrt(bp/bx) when (dose -> inf Gy)
     :params abx: alpha_x / beta_x  [Gy^-1]
     """
 
-    _rbe = (np.sqrt(abx*abx + 4*abx*apx*dose + 4*bpx * dose*dose) - abx) / (2 * dose)
+    _rbe = 1.0 / (2.0 * dose)
+    _rbe[_rbe == np.inf] = 0
```

Comment:
Working with arrays containing np.nan throws tons of errors when plotting, histogramming etc, and causes me to hate pytrip, since things just get more complicated and time consuming to work with. RBE = 0 is totally valid imho. Dose is zero anyway here.